lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -66,5 +70,6 @@
     that:
       - "my_param <= 100"
       - "my_param >= 0"
+    quiet: True
```

Comment:
This specific example says: ` please use 'msg' when ansible version is smaller than 2.7` which means the new argument won't be available. Please create a new example.

Reply:
Yes, my mistake.  Added a new example.